src/test/java/model/CarTest.java

```diff
@@ -0,0 +1,50 @@
+package model;
+
+import common.FakeRandom;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+import java.util.Random;
+
+import static fixture.FakeNumber.MOVE_NUMBER;
+import static fixture.FakeNumber.STOP_NUMBER;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarTest {
+
+    @ParameterizedTest()
+    @DisplayName("OK : 랜덤값이 4 이상일 시 움직임")
+    @ValueSource(ints = {MOVE_NUMBER, 9})
+    void carMove(int fixedValue){
+        Car car = createCar(fixedValue);
+        car.move();
+
+        assertThat(car.getPosition()).isEqualTo(2);
+    }
+
+    @ParameterizedTest()
+    @DisplayName("OK : 랜덤값이 4 미만일 시 움직이지 않음")
+    @ValueSource(ints = {0, STOP_NUMBER})
+    void carNotMove(int fixedValue){
+        Car car = createCar(fixedValue);
+        car.move();
+
+        assertThat(car.getPosition()).isEqualTo(1);
+    }
+
+    @Test
+    @DisplayName("ERROR : 이름 형식이 잘못됨")
+    void InvalidNameCar(){
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new Car("", new Random()));
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new Car(" ", new Random()));
+        Assertions.assertThrows(IllegalArgumentException.class, () -> new Car("test12", new Random()));
+    }
+
+    private Car createCar(int fixedValue){
+        return new Car("test1", new FakeRandom(List.of(fixedValue)));
```

Comment:
파라미터를 바꾸면 FakeRandom를 더 잘 활용할 수 있을 거 같다는 생각이 들었습니다 ! 이렇게 하면, 뭔가 테스트를 더 다양하게 할 수 있을 거 같아요. 가령, A차, B차, C차가 있으면 승자를 예측할 수 있는 테스트도 할 수 있을 거 같아요

카 테스트 보다는 레이싱 테스트에서 사용할 수 있을 거 같아요 !

```suggestion
    private Car createCar(Integer... fixedValue){
        return new Car("test1", new FakeRandom(Arrays.asList(fixedValue)));
```

Reply:
그렇게 한다면 Fake 객체를 더 다양하고 잘 사용할 수 있을 것 같아요!!